src/components/shared/DeviceProtectedRoute.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Route } from 'react-router-dom';
+import LogoImg from '../../images/susi-logo.svg';
+import styled from 'styled-components';
+import _Button from '@material-ui/core/Button';
+
+const Button = styled(_Button)`
+  width: 332px;
+  margin-top: 2rem;
+  @media (max-width: 356px) {
+    width: 100%;
+  }
+`;
+
+const SusiLogo = styled.img.attrs({
+  alt: 'Page Not Found',
+  src: LogoImg,
+})`
+  width: 15rem;
+  margin-bottom: 2rem;
+`;
+
+const Container = styled.div`
+  min-height: 74vh;
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  h2 {
+    font-size: 24px;
+    line-height: 1;
+  }
+`;
+
+const RenderAccessFromSUSI = () => (
+  <Container>
+    <a href="https://susi.ai">
+      <SusiLogo />
+    </a>
+    <h2>Access to this page is restricted from local device,</h2>
+    <h2>try accessing from live website</h2>
+    <a href="https://susi.ai">
+      <Button variant="contained" color="primary">
```

Comment:
Should we pass routeProps to this component and redirect to exact page on susi.ai?

Reply:
Yes, that would be great, on it

Reply:
@fragm3 Also, use `https://susi.ai` from urls.js